Sources/Application/View/View.swift

```diff
@@ -18,28 +18,52 @@ public class View: Viewable {
 		return Int(maxNumberOfGuesses)!
 	}
 
+	public func promptWordSelectionType() {
+		io.display("Would you like a randomly selected word or phrase, or would you like a friend to select one?")
+		io.display("[1] Random")
+		io.display("[2] Select")
+	}
+
+	public func receiveSelectionType(numberOfOptions: Int) -> Int{
```

Comment:
this name seems a bit vague or ambiguous.  Can you think of something better or more descriptive?